backend/tests/SharedResources.Tests/Utf8StringWriter.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.IO;
+using System.Text;
+
+namespace SharedResources.Tests
+{
+    /// <summary>
+    /// Stringwriter that ensures UTF8 is used.
+    /// </summary>
+    public class Utf8StringWriter : StringWriter
```

Comment:
This class is already implemented in AltinnAppGitRepository, but maybe it must be defined again in tests context?

Reply:
It's internal class in designer project.